### PR TITLE
Add funicular and monorail

### PR DESCRIPTION
**Pull request by rmarianski**
@zerebubuth could you review please?

Unfortunately, I couldn't find any route=funicular features in north america so I didn't add a test for it.

### CODE REVIEW

**Comment by rmarianski** [comment]
Connects to #588

**Comment by zerebubuth** [comment]
Does it need a sort key adding in the transit CSV? Also, the new values should be mentioned in the docs.

**Comment by rmarianski** [comment]
Good call on the docs. Updated in a206a2a.

And thanks for catching the `sort_key` too! @nvkelso should these new features be at the same level as `tram` and `light_rail`? ie `415`?

**Comment by nvkelso** [comment]
I've been to these two Pittsburgh "inclines", which were in OSM but missing their relation:

* Duquesne Incline: [OSM relation](http://www.openstreetmap.org/relation/6060405) and [Wikipedia](https://en.wikipedia.org/wiki/Duquesne_Incline)
* Monongahela Incline: [OSM relation](http://www.openstreetmap.org/relation/6060404) and [Wikipedia](https://en.wikipedia.org/wiki/Monongahela_Incline)

**Comment by nvkelso** [comment]
@rmarianski We should probably break them out:

```
kind,sort_key
light_rail,417
tram,416
funicular,415
```

**Comment by rmarianski** [comment]
Updated in 953074f. I added an explicit `sort_key` for `monorail` too, just below `funicular`.

**Comment by rmarianski** [comment]
Thanks for updating those features in osm! Added a test for funicular in 09e0073.

File: spreadsheets/transit.csv [hunk 1]
```diff
@@ -1,5 +1,7 @@
 kind,sort_key
-tram,415
-light_rail,415
+light_rail,417
+tram,416
+funicular,415
+monorail,414
```
Comment (nvkelso) [inline spreadsheets/transit.csv]: This range should start at 415, there's only a couple slots at 414 and below before there are collisions with bridges. The room to grow is up towards 425. So monorail 415, funicular 416, and on up.

**Comment by zerebubuth** [comment]
:+1: looks great now!

File: spreadsheets/transit.csv (hunk 1 above)
Comment (rmarianski) [inline spreadsheets/transit.csv]: Updated in f0090ef.

**Comment by nvkelso** [comment]
LGTM

**Pull request merged**